assets/src/edit-story/components/tabview/index.js

```diff
@@ -228,8 +237,13 @@ function TabView({
   useKeyDownEffect(ref, ['up', 'down'], () => {}, []);
 
   // Home/end keys navigate to first/last tab regardless of direction
-  useKeyDownEffect(ref, 'home', () => selectTabByIndex(0), [selectTabByIndex]);
-  useKeyDownEffect(ref, 'end', () => selectTabByIndex(-1), [selectTabByIndex]);
+  useKeyDownEffect(ref, 'home', () => focusTabByIndex(0), [focusTabByIndex]);
+  useKeyDownEffect(ref, 'end', () => focusTabByIndex(-1), [focusTabByIndex]);
+
+  // Enter/space to select currently focused tab.
+  useKeyDownEffect(ref, ['enter', 'space'], () => selectTab(focused.current), [
```

Comment:
Would this interfere with using space/enter to use the `QuickAction` `+` for adding a text element directly?

Reply:
It did interfere - well spotted! I [wrapped it in a check](https://github.com/google/web-stories-wp/pull/8335/commits/1baf3b3ff6e3abc30223cca478314fcf05063908#diff-d0148c624c999bce1a708f7517bc0393d21e3828c4e696a8d8986dff4c0ddbc3R244-R255) for whether the keyboard focus is on a tab or not (and allowing default if not), and that seems to work great!

Reply:
Yep, works well now 🎉